### DISCUSSION TITLE
Bad magic and different hash

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#1398] [Help]
I would like to know what's the meaning of such warning messages that arise when new version of GO is installed (first run).
It seems to me those messages are telling that chache is bypassed, isn't it ? If true, is it best to delete cahce and recreate it manually ?
tks
![badmagic](https://user-images.githubusercontent.com/50132661/220068882-27838349-dccb-4d5c-83ef-a4c3ad86c967.jpg)
![differenthash](https://user-images.githubusercontent.com/50132661/220068903-eb0fac2f-2741-4962-8f1b-d2cd5e46fffb.jpg)

**Comment by oleg68** [comment]
@paolo2504 The index is rewritten automatically, so you won't see this message next time when you load this organ.

**Comment by thbeckpr** [comment]
That is a confusing, and perhaps unnecessary message. Perhaps it should be suppressed?
